packages/tldraw/src/translations/fr.json

```diff
@@ -3,96 +3,96 @@
   "style.menu.fill": "Remplir",
   "style.menu.dash": "Bordure",
   "style.menu.size": "Taille",
-  "style.menu.keep.open": "Garder ouvert",
-  "style.menu.font": "Font",
+  "style.menu.keep.open": "Toujours afficher le menu",
+  "style.menu.font": "Police",
   "style.menu.align": "Alignement",
   "styles": "Styles",
-  "zoom.in": "Zoomer",
-  "zoom.out": "Dézoomer",
+  "zoom.in": "Zoom avant",
+  "zoom.out": "Zoom arrière",
   "to": "À",
   "to.selection": "Sélection",
-  "to.fit": "Adapter",
+  "to.fit": "Contenu",
   "menu.file": "Fichier",
-  "menu.edit": "Modifier",
-  "menu.view": "Vue",
+  "menu.edit": "Édition",
+  "menu.view": "Afficher",
   "menu.preferences": "Préférences",
   "menu.sign.in": "S'authentifier",
   "menu.sign.out": "Se déconnecter",
   "sponsored": "Sponsorisé",
   "become.a.sponsor": "Devenir un sponsor",
   "zoom.to.content": "Retour au contenu",
-  "zoom.to.selection": "Zoomer sur la sélection",
-  "zoom.to.fit": "Zoomer pour adapter",
-  "zoom.to": "Réinitialiser le zoom à",
-  "preferences.dark.mode": "Mode Sombre",
-  "preferences.focus.mode": "Mode Focus",
-  "preferences.debug.mode": "Débogage Mode",
-  "preferences.show.grid": "Montrer La Grille",
-  "preferences.use.cad.selection": "Utiliser La Sélection CAD",
-  "preferences.keep.stylemenu.open": "Garder Le Menu Style Ouvert",
-  "preferences.always.show.snaps": "Garder Les Snaps Visible",
-  "preferences.rotate.handles": "Manipuler La Rotation",
-  "preferences.binding.handles": "Manipuler La Liaison",
-  "preferences.clone.handles": "Manipuler Le Clonage",
+  "zoom.to.selection": "Ajuster le zoom à la sélection",
+  "zoom.to.fit": "Adapter le zoom au contenu",
+  "zoom.to": "Rétablir le zoom à",
+  "preferences.dark.mode": "Mode sombre",
+  "preferences.focus.mode": "Mode focus",
+  "preferences.debug.mode": "Afficher la barre développeur",
+  "preferences.show.grid": "Afficher la grille",
+  "preferences.use.cad.selection": "Utiliser la sélection CAD",
+  "preferences.keep.stylemenu.open": "Toujours afficher le menu styles",
+  "preferences.always.show.snaps": "Afficher les repères dynamiques",
+  "preferences.rotate.handles": "Manier la rotation",
+  "preferences.binding.handles": "Manier les liaisons",
+  "preferences.clone.handles": "Manier le clonage",
   "undo": "Annuler",
-  "redo": "Refaire",
+  "redo": "Rétablir",
   "cut": "Couper",
   "copy": "Copier",
   "paste": "Coller",
-  "copy.as": "Copier en tant que",
-  "export.as": "Exporter en tant que",
-  "select.all": "Sélectionner tout",
-  "select.none": "Sélectionner aucun",
+  "copy.as": "Copier sous",
+  "export.as": "Exporter sous",
```

Comment:
I think "copier en tant que PNG or exporter en tant que " is better than "Copier sous PNG or Exporter sous" PNG etc... , let's keep that

Reply:
Yes, you are right: "Copier sous PNG" It's bad.
In English "En tant que" would translate to: "As a representative", "to the extent that I am", "as far as I know"... 

I may have a better alternative: "Copier au format" > "PNG" and "Exporter au format" > "PNG/...."
Also, this indicates that we will be able to choose the file format when copying/exporting.

Is this better for you? Or would you prefer that I revert to the original translation?

Reply:
its better thanks

Reply:
Thank @judicaelandria 

It's done!